tests/testthat/test-robustbayesianmetaanalysis.R

```diff
@@ -630,11 +630,11 @@ fittedPath <- file.path("robmaFit.RDS")
     expect_equal_plots(testPlot, "heterogeneity-conditional-truncated-priors", dir="RobustBayesianMetaAnalysis")
   })
   
-  test_that("Forest plot (Conditional) matches", {
-    plotName <- results[["results"]][["plots"]][["collection"]][["plots_theta"]][["data"]]
-    testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
-    expect_equal_plots(testPlot, "forest-plot-conditional-truncated-priors", dir="RobustBayesianMetaAnalysis")
-  })
+#   test_that("Forest plot (Conditional) matches", {
+#     plotName <- results[["results"]][["plots"]][["collection"]][["plots_theta"]][["data"]]
+#     testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
+#     expect_equal_plots(testPlot, "forest-plot-conditional-truncated-priors", dir="RobustBayesianMetaAnalysis")
+#   })
```

Comment:
these and the other commented tests should be skipped instead
```suggestion
   test_that("Forest plot (Conditional) matches", {
      skip(insert a reason for why this fails)
     plotName <- results[["results"]][["plots"]][["collection"]][["plots_theta"]][["data"]]
     testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
     expect_equal_plots(testPlot, "forest-plot-conditional-truncated-priors", dir="RobustBayesianMetaAnalysis")
   })
```

Reply:
fixed